var/spack/repos/builtin/packages/kokkos-tools/package.py

```diff
@@ -10,7 +10,7 @@ class KokkosTools(CMakePackage):
 
     homepage = "https://github.com/kokkos/kokkos-tools/"
     git = "https://github.com/kokkos/kokkos-tools.git"
-
+    maintainers = ("jennfshr", "vlkale", "rbberger")
```

Comment:
```suggestion
    maintainers("jennfshr", "vlkale", "rbberger")
```

Reply:
How did I miss that?  Thanks @rbberger

Reply:
Happens to the best of us. Happy to help :)